src/commandHandlers/bio.ts

```diff
@@ -36,7 +36,13 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
         const data = doc.data();
 
         if (data) {
-            Object.entries((data).bio).forEach(([key, value]) => embed.addField(key.toUpperCase(), value));
+            Object.entries((data).bio).forEach(([key]) => {
+                let value = data.bio[key];
```

Comment:
I think having value from the `forEach` is cleaner `([key, value]`, rather than assigning it manually. Then line `40` is no longer needed?

Reply:
I agree but it won't pass linting.

![Screenshot_20200826_180446](https://user-images.githubusercontent.com/17693494/91368684-f947e200-e7c6-11ea-90dc-ccc52a56d95d.png)

value is type unknown

Reply:
I can test this
```typescript
            Object.entries((data).bio).forEach(([key, value]) => {
                if(key === 'location') {
                    value = data.bio[key].display_name;
                }
                return embed.addField(key.toUpperCase(), value);
            });
```

Reply:
Ah ok, `value` type will be changing 😢

Reply:
Ok leave it how you did it, we can always change it in the future - it is very minor 🤓 . Thank you for trying, but lets keep the linter happen

Reply:
Yes it will work, be javascript won't care, but the linter will not be happy and when we go more strict, will cause the build to fail.

Reply:
No worries. I'll leave it as is.